src/TelnyxRTCClient/TelnyxRTCClient.test.ts

```diff
@@ -23,8 +23,8 @@ Object.defineProperty((global as any).navigator, 'mediaDevices', {
   configurable: true,
 });
 
-describe('VertoClient', () => {
-  const client = new VertoClient({
+describe('TelnyxRTCClient', () => {
```

Comment:
Could you add a spec for the new `module` parameter?

Reply:
Done!